cio/io_test.go

```diff
@@ -52,7 +50,7 @@ func TestNewFIFOSetInDir(t *testing.T) {
 	root := t.TempDir()
 
 	fifos, err := NewFIFOSetInDir(root, "theid", true)
-	assert.NilError(t, err)
+	assert.Nil(t, err)
```

Comment:
Testify has an error-specific assertion that can be used that provides a slightly more tailored failure message.

You can use this in a bunch of places (many of the uses of `assert.Nil` are related to error-checking).

```suggestion
	assert.NoError(t, err)
```